api/container_delete.go

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Sorry, I should have specified, but I think HashiCorp style is to not include per file license info, everything is covered by LICENSE in the root directory

Reply:
done